libs/editor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -1583,5 +1583,8 @@ public void onGutenbergDialogNegativeClicked(@NonNull String instanceTag) {
     @Override
     public void onConnectionStatusChange(boolean isConnected) {
         getGutenbergContainerFragment().onConnectionStatusChange(isConnected);
+        if (BuildConfig.DEBUG && hasFailedMediaUploads()) {
```

Comment:
```suggestion
        if (BuildConfig.DEBUG && isConnected && hasFailedMediaUploads()) {
```
I would suggest adding an extra `isConnected` condition is it is available in this scope though it might not be necessary.

Reply:
Ooh! A very good suggestion. Thank you.